bridge/client/app/project-board/project-board.component.ts

```diff
@@ -16,51 +17,57 @@ export class ProjectBoardComponent implements OnInit, OnDestroy {
   private readonly unsubscribe$ = new Subject<void>();
 
   public logoInvertedUrl = environment?.config?.logoInvertedUrl;
-  public project$: Observable<Project | undefined>;
+  public projectName$: Observable<string>;
+  public hasProject$: Observable<boolean | undefined>;
   public contextId?: string;
-  private readonly _projectTimerInterval = 30_000;
   private _errorSubject: BehaviorSubject<string | undefined> = new BehaviorSubject<string | undefined>(undefined);
   public error$: Observable<string | undefined> = this._errorSubject.asObservable();
   public isCreateMode$: Observable<boolean>;
 
-  constructor(private router: Router, private route: ActivatedRoute, private dataService: DataService) {
-    const projectName$ = this.route.paramMap.pipe(
+  constructor(private router: Router, private route: ActivatedRoute, public dataService: DataService, @Inject(INITIAL_DELAY_MILLIS) private initialDelayMillis: number) {
```

Comment:
What's the reason behind `public` here?

Reply:
A leftover from tryouts for testing. I will set it to private again.